docs/advanced-use/edge-proxy.md

```diff
@@ -34,34 +62,36 @@ access to it.
 
 :::
 
+### With docker run:
+
 ```bash
 # Download the Docker Image
 docker pull flagsmith/edge-proxy
 
 # Run it
 docker run \
-    -e FLAGSMITH_API_URL='https://api.flagsmith.com/api/v1/' \
-    -e FLAGSMITH_API_TOKEN='<API Token - see below>' \
-    -e ENVIRONMENT_API_KEYS='<Flagsmith Environment Key - see below>' \
+    -v /<path-to-local>/config.json:/app/config.json \
     -p 8000:8000 \
     flagsmith/edge-proxy:latest
 ```
 
-The Proxy is now running and available on port 8000.
-
-### Environment Variables
-
-You can configure the Edge Proxy with the following Environment Variables:
+### With docker compose:
+
+```yml
+version: '3.9'
+services:
+ edge_proxy:
+  image: flagsmith/edge-proxy:latest
+  volumes:
+   - type: bind
+     source: ./config.json
+     target: /app/config.json
+  ports:
+   - target: 8000
+   - published: 8000
+```
 
-- `FLAGSMITH_API_URL` **Required**. The URL of the API to proxy e.g. `https://api.flagsmith.com/api/v1/` or your own
-  domain name if you are self hosting the Flagsmith API.
-- `ENVIRONMENT_API_KEYS` **Required**. A list of Environment keys to serve. Provided as a comma-separated String e.g.
-  `4vfqhypYjcPoGGu8ByrBaj, EmJFz265Q6CAXuGRZYnkeE, "8KzETdDeMY7xkqkSkY3Gsg`
-- `FLAGSMITH_API_TOKEN` **Required**. Can be retrieved as per [these instructions](/clients/rest.md##private-endpoints).
-- `API_POLL_FREQUENCY` set in seconds. The number of seconds to wait before polling `FLAGSMITH_API_URL` to update the
-  Environments currently in use. Defaults to `10` seconds.
-- `WEB_CONCURRENCY` The number of [Uvicorn](https://www.uvicorn.org/) workers. Defaults to `1`. Set to the number of
```

Comment:
What happened to the other env vars like web concurrency?

Reply:
Ah, yes, only `WEB_CONCURRENCY` is used now. Rest of them are now part of json configuration.